portfolio/src/main/webapp/script.js

```diff
@@ -29,15 +29,22 @@ function addRandomGreeting() {
 
 function goToData() {
     const responsePromise = fetch('/data');
+    responsePromise.then(responsePromise => responsePromise.json());
```

Comment:
You don't need to run `then` on responsePromise twice. (So you'll need to delete one of the lines...can you figure out which line is the correct one?)